docs/mounting.md

```diff
@@ -7,9 +7,26 @@
         gcloud auth login
         gcloud auth list
 
-   Alternatively, you can authenticate Cloud Storage FUSE by setting the ```--key-file``` flag to the path of a JSON key file, which you can download from the Google Cloud console. You can also set the ```GOOGLE_APPLICATION_CREDENTIALS``` environment variable to the path of the JSON key.
+   Alternatively, you can authenticate Cloud Storage FUSE using a service account
+key.
 
-        GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json gcsfuse [...]
+   - [Create a service account](https://cloud.google.com/iam/docs/service-accounts-create)
+  with one of the following roles:
+
+     * `Storage Object Viewer (roles/storage.objectViewer)` role to mount a
+    bucket with read-only permissions.
+     * `Storage Object Admin (roles/storage.objectAdmin)` role to mount a bucket with read-write permissions.
```

Comment:
I think all permissions is good?

Reply:
I believe yes, PTAL at Testing details in the PR description.